README.md

```diff
@@ -36,6 +36,17 @@ producing complex, multi-container applications.
    * `docker` is default service and does not require above configuration.
    * Enable multiple services as comma separated list. Eg: 'docker, openshift'
 
+4. Enable any specific options for the services you have selected:
+
+    * OpenShift
+
+        Specific versions can be specified using the following variables:
+
+        `config.servicemanager.openshift.docker_registry = "docker.io"` - What registry should be pulled from
```

Comment:
@hferentschik : The feature is being implemented in chunks with testing. ATM there is no code to take care of it, but README is our TODO to look at and implement. :-)